sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -339,32 +341,94 @@ async function main() {
 main();
 ```
 
-### List all versions of a certificate
+### Getting the full information of a certificate
 
-`listPropertiesOfCertificateVersions` will list versions of the given certificate.
+Azure's KeyVault's design makes sharp distinctions between Keys,
+Secrets and Certificates. The KeyVault service's Certificates
+features were designed making use of it's Keys and Secrets capabilities.
+Let's evaluate the composition of a KeyVault Certificate:
 
-```javascript
-const { DefaultAzureCredential } = require("@azure/identity");
-const { CertificateClient } = require("@azure/keyvault-certificates");
+> When a Key Vault certificate is created, an addressable key
+> and secret are also created with the same name. The Key Vault
+> key allows key operations and the Key Vault secret allows retrieval
+> of the certificate value as a secret. A Key Vault certificate
+> also contains public x509 certificate metadata.  
+> _Source: [Composition of a Certificate][Composition-of-a-Certificate]._
 
-const credential = new DefaultAzureCredential();
+Knowing that the private key is stored in a KeyVault Secret,
+with the public certificate included, we can retrieve it
+by using the [KeyVault Secrets client][KeyVault-Secrets-client].
 
-const vaultName = "<YOUR KEYVAULT NAME>";
-const url = `https://${vaultName}.vault.azure.net`;
+```ts
+// Using the same credential object we used before,
+// and the same keyVaultUrl,
+// let's create a SecretClient
+const secretClient = new SecretClient(keyVaultUrl, credential);
 
-const client = new CertificateClient(url, credential);
+// Assuming you've already created a KeyVault certificate,
+// and that certificateName contains the name of your certificate
+const certificateSecret = await secretClient.getSecret(certificateName);
 
-const certificateName = "MyCertificateName";
+// Here we can find both the private key and the public certificate, in PKCS 12 format:
+const PKCS12Certificate = certificateSecret.value!;
 
-async function main() {
-  for await (let certificateProperties of client.listPropertiesOfCertificateVersions(certificateName)) {
-    console.log("version: ", certificateProperties.version);
-  }
-}
+// You can write this into a file:
+fs.writeFileSync("myCertificate.p12", PKCS12Certificate);
+```
 
-main();
+Note that, by default, the content type of the certificates
+is [PKCS 12][PKCS_12]. By specifying the content type
+of your certificate, you'll be able to retrieve it in PEM format.
+Before showing how to create PEM certificates,
+let's first explore how to retrieve a PEM secret key
+from a PKCS 12 certificate first.
+
+Using `openssl`, you can retrieve the public certificate in
+PEM format by using the following command:
+
+    openssl pkcs12 -in myCertificate.p12 -out myCertificate.crt.pem -clcerts -nokeys
```

Comment:
Should be a code block using triple backticks. Doesn't need a language i.e. can just be preformatted text.